go-controller/pkg/ovn/namespace.go

```diff
@@ -228,16 +229,19 @@ func (oc *Controller) configureNamespace(nsInfo *namespaceInfo, ns *kapi.Namespa
 	// created
 
 	// If multicast enabled, adds all current pods in the namespace to the allow policy
-	oc.multicastUpdateNamespace(ns, nsInfo)
+	if err := oc.multicastUpdateNamespace(ns, nsInfo); err != nil {
+		klog.Error(err.Error())
```

Comment:
Maybe it makes sense to make `configureNamespace` return an error? It can be propagated by `ensureNamespaceLocked`